src/utils.ts

```diff
@@ -9,3 +9,15 @@ export const isStaff = (member: GuildMember | null | undefined) => {
     staffRoles.includes(role.name.toLowerCase())
   );
 };
+
+//Discords limit for messagelength
```

Comment:
just gonna
```suggestion
// Discord's limit for message length
```
real quick